compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -1274,9 +1274,30 @@ trait ParallelTesting extends RunnerOrchestration { self =>
       case None => files
     }
 
+    class JointCompilationSourceFromTasty(
+       name: String,
+       file: JFile,
+       flags: TestFlags,
+       outDir: JFile,
+       fromTasty: Boolean = false,
+       decompilation: Boolean = false
+    ) extends JointCompilationSource(name, Array(file), flags, outDir, fromTasty, decompilation) {
+
+      override def buildInstructions(errors: Int, warnings: Int): String = {
+        s"""|
+            |Test '$title' compiled with $errors error(s) and $warnings warning(s),
+            |the test can be reproduced by running:
+            |
+            |  sbt "testFromTasty $file"
```

Comment:
This is the easiest way to reproduce but is it helpful for debugging? My experience is that when you run a test with vulpix and the compiler crashes then the exception is swallowed and I either need to look at the test log or run the compiler manually.

I often want to run the compiler manually outside vulpix and I would rather have the manual instructions. E.g:
```
> dotc -d tmp Foo.scala -some-options
> dotc Foo -classpath tmp -from-tasty -some-options
```